### PR TITLE
support window resize , muitl map mode

### PR DESCRIPTION
**Pull request by sunqirui1987**

### CODE REVIEW

File: game.go [hunk 1]
```diff
@@ -300,6 +301,20 @@ type cameraConfig struct {
...
+const (
+	mapModelDefault mapModelEnum = iota
+	mapModelRepeat
+	mapModelFill
+	mapModelFillRatio
+	mapModelFillCut
+)
+
+type mapConfig struct {
+	Model mapModelEnum `json:"model"`
```
Comment (xushiwei) [inline game.go:315]: is mode, not model

File: test/Bananas/res/index.json [hunk 2]
```diff
@@ -8,6 +8,9 @@
       "y": 180
     }
   ],
+  "map":{
+    "model":2
```
Comment (xushiwei) [inline test/Bananas/res/index.json:12]: 2 is unreadable, can be a string.

Comment (sunqirui1987) [reply test/Bananas/res/index.json]: fixed

File: game.go (hunk 1 above)
Comment (sunqirui1987) [reply game.go]: ok

File: test/MiniMapCamera/res/index.json [hunk 3]
```diff
@@ -11,6 +11,9 @@
   "camera":{
     "on":"Monkey"
   },
+  "map":{
+    "mode":1
```
Comment (xushiwei) [inline test/MiniMapCamera/res/index.json:15]: 1 is forgotten to fix?

Comment (sunqirui1987) [reply test/MiniMapCamera/res/index.json]: fixed

**Pull request merged**